src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -307,16 +307,17 @@ const ExampleObjectLinks = (props: BrowserProps) => {
   if (!activeGenomeId) {
     return null;
   }
+  console.log('ExampleObjectLinks', props);
```

Comment:
Can this be removed?

Reply:
Removed the console.log.

Also (because I forgot to do that before) I removed the `Spliced mRNA length` field from `BrowserBar` and updated the tests for BrowserBar.